app/views/catalog/record/_marc_contents_summary.html.erb

```diff
@@ -42,8 +42,8 @@
   </dd>
 <% end %>
 
-<% summary = document.fetch(:summary_struct, []).first %>
-<% if summary.present? %>
+<% summary = document.fetch(:summary_struct, []).first || {} %>
```

Comment:
I assume `summary_struct` is an enumerable that returns hash-like objects?

Reply:
Yeah -- long story short, we push some of the MARC parsing into solr so we don't need to parse MARC records from the `catalog#index` page.